style/elixir/README.md

```diff
@@ -3,9 +3,91 @@ Elixir
 
 [Sample](sample.ex)
 
+* Don't use a space for padding for a tuple with two or less elements. `{1, 2}`
+* Don't indent pipes when not using the match operator (`=`).
 * Don't separate multiple declarations of functions of the same name with
+* Don't use a space for padding for a list with two or less elements. `[1, 2]`
   newlines.
-* Indent pipes by two spaces when using the match operator (`=`).
-* Don't indent pipes when not using the match operator (`=`).
+* Don't put spaces around matched pairs like brackets, parentheses, etc.
+* Don't use spaces after non-word operators that only take one argument; or 
+  around the range operator.
+* Don't put a space between a function name and the opening parenthesis.
+* Don't use parentheses in macro calls when a do block is passed.
+* Don't prefix macros that return boolean for guards with `is_`
+* Don't put a blank line after defmodule.
+* Don't explicitly set nil for a structs attribute just use a list of atoms.
+* Avoid binding data to a variable when you can wrap it in a function and 
+  thoughtfully name it after it's intent.
+* Avoid binding data to a variable when not in a tuple, just to pass it to 
+  another function, use `with` instead.
+* Avoid using the pipe operator just once.
```

Comment:
I don't think this is necessary. There are sometimes it's clearer to use a pipe. I vote for avoiding a rule here.